Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Runtime.ExceptionServices;
+using System;
+using System.Reflection.Metadata.Ecma335;
+
+namespace BootCamp.Chapter
+{
+    public class Inventory
+    {
+        protected Item[] _items;
```

Comment:
I believe my initial rationale for that was so that the _items can be accessed within child classes through inheritance but not outside of that. I have since changed thing when implementing the change for the comment below.

Reply:
In general, inheritance is a poor choice of disign. In some cases it does make sense, in other cases not so much. I'd say make it private, because for sure in this homework you won't need it protected (inherited).

Reply:
I agree. The changes for this were made within the latest commit. Please let me know if you find any more things that require changing. Many thanks in advance!

Reply:
I have just realised that it is currently public rather than private. Rationale is that the although it would be correct to have it private, due to me having separated Shop and Player inventory classes that inherit from base (Inventory), refactoring would take too long. Hope this isnt an issue :)